subiquity/common/filesystem/manipulator.py

```diff
@@ -48,15 +48,15 @@ def create_filesystem(self, volume, spec):
         if spec.get('fstype') is None:
             # prep partitions are always wiped (and never have a filesystem)
             if getattr(volume, 'flag', None) != 'prep':
-                volume.wipe = None
+                volume.wipe = spec.get('wipe', None)
             fstype = volume.original_fstype()
             if fstype is None:
                 return None
-            preserve = True
+            volume.preserve = preserve = volume.wipe is None
```

Comment:
Ah yeah, not sure about this change?

Reply:
I believe the new code is correct.  If you still have concerns then we may have a different vision for this function and should discuss..